source/mainnet/smart-contracts/tutorials/wCCD/wCCD-front-end-set-up.rst

```diff
@@ -137,104 +127,6 @@ part of the Concordium blockchain.
-            Add the following port forwarding rule to your method to ssh into your instance by running the command in another terminal.
-            The port 10001 on your localhost is forwarded to the port 10001 on your instance.
-
-            .. code-block:: console
-
-                $ssh -NL localhost:10001:<IP-address-of-your-instance>:10001 <username>@<host>
-
-        If you don't have a running testnet node or port forwarding enabled, :ref:`Setup the development environment<setup-env>`
-        will guide you through these setup steps.
-
-        These prerequisites ensure that you have a testnet node reachable locally on port 10001. The |bw| requires the
-        `Concordium JSON-RPC server <https://github.com/Concordium/concordium-json-rpc>`_
-        that points to your node. Clone the repository with the command:
-
-        .. code-block:: console
-
-            $git clone https://github.com/Concordium/concordium-json-rpc.git
-
-        Build and run the server as described in the README file of the
-        `JSON-RPC repository <https://github.com/Concordium/concordium-json-rpc>`_.
-
-        The final command that you execute to start the ``JSON-RPC server`` is as follows:
-
-        .. code-block:: console
-
-            $yarn start --port 9095 --nodeAddress 127.0.0.1 --nodePort 10001 --nodeTimeout 5000
-
-        Your ``JSON-RPC server`` is running on port `9095` and connects to your local node on port `10001`.
-        You are ready now to build the |bw| from the source code.
-        Clone the repository with the command:
-
-        .. code-block:: console
-
-            $git clone https://github.com/Concordium/concordium-browser-wallet.git
-
-        You can build the browser wallet extension and load the ``dist`` folder
-        (located at ``root/packages/browser-wallet/dist``) into the
-        `Chrome browser <https://developer.chrome.com/docs/extensions/mv3/getstarted/#unpacked>`_ as
-        described in the README file of the
-        `browser wallet repository <https://github.com/Concordium/concordium-browser-wallet/tree/main/packages/browser-wallet>`_.
-
-        .. Note::
-
-            Depending on the exact commit hash that you used to build your |bw|, the screenshots and setup steps might differ.
-
-        The next steps are based on the pre-MVP |bw| from a git commit before 22.8.2022. These early versions of the pre-MVP |bw| have an input field for the private key to import an already existing account and an input field for the link to the JSON-RPC server. Alternatively, you can follow workflow 1 to use the most recent |bw| without hosting your own node and JSON-RPC server. The |bw| from workflow 1 can be used to create a new account (no importing of private keys needed).
-
-        You are ready now to start the |bw| by clicking on the Concordium icon at the top right of the
-        ``Chrome`` browser.
-
-        .. image:: ./images/wCCD_tutorial_12.png
```

Comment:
This picture can be deleted out of the folder `./images/wCCD_tutorial_12.png` since it is not used anymore.

Reply:
Remvoed.